importDBdmpFile/dropGocdbUser2.sh

```diff
@@ -19,9 +19,8 @@
 #
 # RESULT=$(sqlplus system/XXXXXXX <<- ENDSQL
 
-# PASSFILE is in "parfile" format for impdp
-PASSFILE=/root/gocdb-failover-scripts/importDBdmpFile/pass_file
-USER_PASS=$(cat "${PASSFILE}" | grep "^userid=" | cut -d "=" -f 2)
+# pass_file is in "parfile" format for impdp
+USER_PASS=$(cat "pass_file" | grep "^userid=" | cut -d "=" -f 2)
 
 # Need to exit gracefully if pass_file doesn't exist or
```

Comment:
Is this implemented? Could add check to fail also if permissions on PASSFILE are too open.

Reply:
One can only hope :p. Feel like opening an issue about adding a check on pass_file's permissions?

Reply:
Thanks for opening #5 😄